### PR TITLE
Feature structured output

### PR DESCRIPTION
**Pull request by flbulgarelli**
Requires merging mumuki/mumukit#20 before

### CODE REVIEW

**Comment by flbulgarelli** [comment]
Merge after #8

File: Gemfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 source 'https://rubygems.org'
 
 gem 'rake'
-gem 'mumukit', github: 'uqbar-project/mumukit', tag: 'v0.2.0'
+gem 'mumukit', github: 'uqbar-project/mumukit', branch: 'master'
 gem 'rspec', '2.13'
 
 group :test do
-  gem 'mumukit-bridge', github: 'mumuki/mumukit-bridge', tag: 'v0.1.0'
+  gem 'mumukit-bridge', github: 'mumuki/mumukit-bridge', branch: 'master'
```
Comment (faloi) [inline Gemfile]: Don't forget to update these versions when released

File: spec/integration_spec.rb [hunk 2]
```diff
@@ -1,33 +1,59 @@
+require_relative '../spec/spec_helper' #FIXME remove. There must be an issue with mumukit-bridge requires
+
 require 'mumukit/bridge'
 
 describe 'runner' do
   let(:bridge) { Mumukit::Bridge::Bridge.new('http://localhost:4567') }
 
   before(:all) do
-    @pid = Process.spawn 'rackup -p 4567 > /dev/null 2>&1'
+    @pid = Process.spawn 'rackup -p 4567', err: '/dev/null'
```
Comment (faloi) [inline spec/integration_spec.rb]: Much nicer than the creepy `2>&1` :smile_cat:

File: lib/test_runner.rb [hunk 3]
```diff
@@ -1,12 +1,27 @@
...
+  end
+
+  def post_process_file(file, result, status)
+    [transform(JSON.parse(result)['examples'])]
+  rescue JSON::ParserError
+    [result, :errored]
   end
 
+  def transform(examples)
+    examples.map { |e| [e['full_description'], e['status'].to_sym, parse_out(e['exception'])] }
```
Comment (faloi) [inline lib/test_runner.rb]: The possible values of RSpec's `status` field directly match our own?

Comment (flbulgarelli) [inline lib/test_runner.rb]: Yes, they are passed/failed, exactly as our's

File: Gemfile (hunk 1 above)
Comment (flbulgarelli) [inline Gemfile]: Yes. We have to update mumukit and bridge in platform and runners

**Pull request merged**